derpconf/config.py

```diff
@@ -76,6 +76,16 @@ def load(cls, path, conf_name=None, lookup_paths=[], defaults={}):
 
     @classmethod
     def __load_from_path(cls, conf, path):
+        if isdir(path):
+            conf.config_folder = path
+            files = sorted(os.listdir(path))
```

Comment:
I think that's artifically limiting, we don't know how people might want to use it or what naming conventions they'd pick for their files. The same feature can be found in a lot of services and the config folders are assumed to only contain config files.